Documentation/devel/distribution-point.md

```diff
@@ -0,0 +1,174 @@
+# Distribution points 
+
+A *distribution point* represents a method of fetching an image from an input string. This does not mandate an images type. Usually a distribution point can only provide one image type but some can provide different image types. For example:
+
+* Some *Docker* registries also provide *OCI images*.
```

Comment:
I went ahead and turned this Paragraph + Bullets into two paragraphs for flow and paragraph conciseness.  Let me know if it ought to just be one paragraph.